website/docs/d/paas.html.md

```diff
@@ -42,7 +42,7 @@ The following attributes are exported:
   * `name` - Name of a port.
   * `listen_port` - Port number.
 * `security_zone_uuid` - The UUID of the security zone that the service is attached to.
-* `network_uuid` - Network UUID containing security zone.
+* `network_uuid` -  The UUID of the network that the service is attached to or network UUID containing security zone.
```

Comment:
`network_uuid` is the private network UUID. It's not related to the security zone.